privacyidea/lib/importotp.py

```diff
@@ -568,7 +568,7 @@ def decrypt(self, input_data):
                 decrypted.status, decrypted.stderr))
             raise Exception(decrypted.stderr)
 
-        return decrypted.data
+        return to_unicode(decrypted.data)
```

Comment:
Originally I wanted to note that we can't be sure that we always decrypt valid UTF-8 data. However, I noticed we only use ``GPGImport`` for importing token files which we can expect to be UTF-8. So let's keep it this way, could you just add a note to the docstring?

Reply:
my thoughts exactly :-) I will update the docstring.

Reply:
fixed in 21bdef5b